### PR TITLE
More IAA candy

### PR DESCRIPTION
**Pull request by JMWTurner**
- Briefcases and Secure Briefcases are now empty by default
- IAA now spawn with a Nanotrasen themed briefcase. Only those spawn with the encryption keys for commendations and demotions

briefcent

### CODE REVIEW

**Comment by Kurfursten** [comment]
First impression: that looks like a horrifying monster suitcase with two glowing eyes and monstrous teeth.

**Comment by JMWTurner** [comment]
Well if it gives IAA an intimidation factor then it's all the better.

**Comment by gbasood** [comment]
The arrows around the N might be smaller to make it look more like arrows maybe??
I'd like to see what that'd look like at least. I know they're already pretty small though.

**Comment by JMWTurner** [comment]
> The arrows around the N might be smaller to make it look more like arrows maybe??

Well I'm using 3 pixels per arrow so good luck with that.

File: code/game/objects/items/weapons/storage/secure.dm [hunk 1]
```diff
@@ -228,7 +228,7 @@
 //        Secure Safe
 // -----------------------------
 
-/obj/item/weapon/storage/secure/safe
+/obj/item/weapon/storage/secure/safe//Why the fuck isn't this a structure
```
Comment (ComicIronic) [inline code/game/objects/items/weapons/storage/secure.dm]: No meme comments

Comment (JMWTurner) [reply code/game/objects/items/weapons/storage/secure.dm]: Why isn't it a structure though? If possible I'd rather turn it into a structure here and now unless someone gives me a good enough reason.

Comment (ComicIronic) [reply code/game/objects/items/weapons/storage/secure.dm]: Because oldcoders were badcoders, but you can't change it in this PR. No meme comments

Comment (JMWTurner) [reply code/game/objects/items/weapons/storage/secure.dm]: aye

**Comment by ComicIronic** [comment]
The only minor issue I can see with this is that the Lawyer traitor item is a briefcase, so if the IAA orders it, it might look fishy, but that won't block this.

**Comment by Kurfursten** [comment]
@ComicIronic That's actually a good point. Moreover, all alt-titles get the new briefcase instead of just IAAs.

**Comment by Durgen** [comment]
maybe the traitor item should use this sprite

**Comment by JMWTurner** [comment]
No, there are other regular suitcases in the IAA office. On top of that, more than one centcom briefcase per IAA WOULD be suspicious.

**Comment by Shadowmech88** [comment]
What about making it so that the gun can be removed from the gun-briefcase, turning it into a regular briefcase, and the IAA can then put that gun into their NT briefcase if they wish, turning that into a gun-briefcase?

**Comment by JMWTurner** [comment]
@Shadowmech88 You can already place the SMG into the NT briefcase, I did it yesterday in fact when I was tator IAA. People rarely if ever check what contains your briefcase, and it's better than having to carry a screwdriver everywhere.

**Comment by Kurfursten** [comment]
You don't need to carry a screwdriver, the most powerful part about the falsebottom briefcase is that you can shoot with the briefcase. You don't have to remove the SMG.

**Comment by JMWTurner** [comment]
ohhh. If only I had known that yesterday. That'd have been so cool. Whatever.

In that case I suggest we rather add an item that let's you turn any briefcase into a false-bottomed one. Or stickers that let you disguise any briefcase into a nanotrasen themed one.

**Comment by StrikS** [comment]
Thank you for that amazing sprite. IAA has become 20% cooler. ^^

**Comment by JMWTurner** [comment]
no problem :)

**Comment by Exxion** [comment]
Oh god please no

**Pull request merged**